### ISSUE TITLE
wrapped video control not displaying/playing content

### ISSUE THREAD

**Issue by tkntobfrk** [SplitmediaLabsLimited/xui#4]
Are there any restrictions on html markup. I have a wrapped video control basicly like the code below which I plan on using before updating the rtmp stream. Sort of a preview function.

  <div class="card">
            <div class="video-container padding-top">
              <video controls width="560" height="315" ng-src="{{trustSrc(o.mediaWebCDN)}}">
              </video>
            </div>
     </div>

This works fine straight through the browser with no errors in the console, but when run through tools-->script plugins content appears to be blank and will not play.
  
This amazon cloudfront in S3 with a web distribution which this uses, the rtmp stream is what I send to xsplit.

**Comment by dcefram** [comment]
There should be no restrictions on the HTML markup. What version of XSplit Broadcaster are you using? What I'm suspecting is that the video tag is not supported on the Broadcaster version that you are currently using (older version of Broadcaster most likely would be using an older version of chromium, which could possibly not support some HTML5 elements)

**Comment by tkntobfrk** [comment]
2.5.1509.0807

Using that as a plugin does work on my side.

However creating an page locally seems to not work.

    <video controls>
      <source src="somevideo url" type="video/mp4">
    </video>

I'll give this a better look when I have a chance. Video tags are always so hit and miss.

**Comment by dcefram** [comment]
Right, we'll also look into that on the latest developer version of XSplit. One thing that I tried which is consistent on not loading is when using video tags for WebRTC.

**Comment by tkntobfrk** [comment]
I actually looked into this a bit when researching another item related to xsplit controls through the script. Its more or less because CEF doesn't have a various items fully implemented. I was actually looking to see if I could use chromiums webrtc text to speech, but after digging around it doesnt appear implemented in CEF.  For my purposes I dont need this portion run through the xsplit plugin control. So I run it through chrome,upload the wav to s3, convert it the an mp3 via lambda, then play the recording via the rtmp plugin. Works fine just a lot of hoops.

**Comment by dcefram** [comment]
Oh, if you were trying to play wav files or mp3/4 files on CEF, it wont work. The reason behind that is due to those extensions are proprietary, and are subject for licensing. CEF does not support those extension out of the box.

Read: https://code.google.com/p/chromiumembedded/issues/detail?id=371

**Comment by tkntobfrk** [comment]
Well I was trying to record audio input file via webrtc, its all the same though lots of unsupported parts
